assets/src/edit-story/app/layout/layout.js

```diff
@@ -66,6 +68,7 @@ const Area = styled.div`
   z-index: 2;
 `;
 
+// TODO: Fix meta boxes layout.
```

Comment:
I think the current grid layout is _OK_. Can ask Sam in follow-up tickets for UX.